internal/parser.go

```diff
@@ -77,7 +76,7 @@ func NewParser(cfg string, opts *Options, fs FileSystem) Parser {
 	return GOBDeserialize(pStr, &p)
 }
 
-// Do the parsing process or skip if cached.
+//Bootstrap Do the parsing process or skip if cached.
```

Comment:
What's the reason to add the method name within the comment? Also, there should be a space after `//`